cylc/flow/network/schema.py

```diff
@@ -309,7 +311,7 @@ class SortArgs(InputObjectType):
     'exids': List(ID, default_value=[]),
     'states': List(String, default_value=[]),
     'exstates': List(String, default_value=[]),
-    'is_held': Boolean(),
+    'is_queued': Boolean(),
```

Comment:
@hjoliver the GitHub UI shows `NODES_EDGES_ARGS` with `is_held` **and new `is_queued`. But `NODES_EDGES_ARGS_ALL` with `is_queued` replacing `is_held`. Is that correct?

Reply:
good spotting again :1st_place_medal: that's an error, supposed to have both in the schema.